Doc/tutorial/inputoutput.rst

```diff
@@ -133,7 +133,14 @@ applies :func:`repr`::
    >>> print(f'My hovercraft is full of {animals!r}.')
    My hovercraft is full of 'eels'.
 
-For a reference on these format specifications, see
+The ``=`` specifier can be used to expand an expression and its evaluation:
```

Comment:
Shouldn't it say something about the text of the expression? "Expand an expression" doesn't sound like that, to me.

This should also mention that it's the repr that's printed.

Reply:
I did try to condense the wording a bit for the tutorial and just rely more on the example, but on a second read through you're definitely right that it sounds off. If borrowing the wording from the what's new,

The `=` specifier can be used to expand an expression to the text of the expression, an equal sign, then the representation of the evaluated expression.

leaving out some information gives,

The `=` specifier can be used to expand an expression to itself and the representation of its evaluation.

Would that work, or would you prefer the original wording? I'm also not too sure what the difference is between the 'representation of an evaluation' vs just 'evaluation'. Say for `i=1`, would the representation be `i.__repr__()` and the evaluation `eval('i')`?